Widgets/LeaderBoard.php

```diff
@@ -13,6 +14,7 @@ class LeaderBoard extends Widget
     public function run()
     {
         $now = Carbon::now();
```

Comment:
Why not `$now = Carbon::now()->locale(config('app.locale'));` ?

Locale is already stored in config, no need to load the App facade again for getting a simple string value.

Reply:
And even we use it for just month names here, we should test this to be sure that `now()` returns the UTC datetime but only changes the locale, so there should be no timezone change when locale changes 😉

Reply:
> Why not `$now = Carbon::now()->locale(config('app.locale'));` ?
> 
> Locale is already stored in config, no need to load the App facade again for getting a simple string value.

Oupss it's right